linux_os/guide/system/permissions/mounting/grub2_nousb_argument/oval/shared.xml

```diff
@@ -1,34 +1,26 @@
+{{# Enable this line when supported_platforms option is available
+{{{ oval_grub_config(parameter="GRUB_CMDLINE_LINUX", value="nousb", supported_platforms=['rhel7', 'rhel8', 'fedora']) }}}
+#}}
+{{% set path = "/etc/default/grub" %}}
+{{% set parameter = "GRUB_CMDLINE_LINUX" %}}
+{{% set value = "nousb" %}}
 <def-group>
-  <definition class="compliance" id="grub2_nousb_argument.rule" version="1">
+  <definition class="compliance" id="{{{ rule_id }}}" version="1">
     <metadata>
-      <title>Disable Kernel Support for USB via Bootloader Configuration</title>
+      <title>{{{ rule_title }}}</title>
       <affected family="unix">
         <platform>Red Hat Enterprise Linux 7</platform>
         <platform>Red Hat Enterprise Linux 8</platform>
         <platform>multi_platform_fedora</platform>
       </affected>
-      <description>Look for 'nousb' argument in the kernel line in /etc/default/grub</description>
+      <description>{{{ rule_description }}}</description>
```

Comment:
This should be: `Look for parameter '{{{ parameter }}}' with value '{{{ value }}}' in '{{{ path }}}'`

Reply:
Thanks for the catch. I'll also update fix the jenkins problem.